sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -16,7 +16,9 @@
 - `USReceiptType` is renamed to `ReceiptType`
 - `use_training_labels` is now a required positional param in the `begin_training` APIs.
 - `stream` and `url` parameters found on methods for `FormRecognizerClient` have been renamed to `form` and `form_url`, respectively.
-For recognize receipt methods, parameters have been renamed to `receipt` and `receipt_url`.
+- For recognize receipt methods, parameters have been renamed to `receipt` and `receipt_url`.
```

Comment:
Should also update the readme, i.e. here (i think there are two code snippets that need updating): https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/README.md#train-a-model